### ISSUE TITLE
@techreport type of bibitem

### ISSUE THREAD

**Issue by MikhailLipanin** [yegor256/bibcop#7]
Now, it is just removes `@techreport`-s and gets this warning:
```
I don't know what to do with @techreport type of bibitem
```

**Comment by MikhailLipanin** [comment]
same with `@incollection`
![image](https://user-images.githubusercontent.com/95661085/207093167-c3b618af-b87c-4aa5-acd7-f8cb39de8aca.png)
